test/functional/crud_spec_tests.js

```diff
@@ -14,6 +15,7 @@ function findScenarios(type) {
 
 const readScenarios = findScenarios('read');
 const writeScenarios = findScenarios('write');
+const dbScenarios = findScenarios('db');
```

Comment:
I'd like to hold off here until we can move the db-level aggregate into the `read` folder. We shouldn't run a whole separate tests suite for just two tests